### PR TITLE
Sort third-party packages and add `starlette-wtf`

### PR DESCRIPTION
**Pull request by aminalaee**
According to discussion comment [here](https://github.com/encode/starlette/discussions/1411#discussioncomment-1968621) adds starlette-wtf package to the docs.

### CODE REVIEW

**Comment by aminalaee** [comment]
I can see the third-party packages can be improved. We can sort the packages alphabetically, the same as Third Party Middlewares, and move the middleware to Third Party Middlewares.

Maybe even include #1414 here as we'll move a few things around.

**Comment by aminalaee** [comment]
Right now we have `Starlette Context`, `Starlette Cramjam` and `Starlette OAuth2 API` in third party packages but they're middleware and should be moved to Third Party Middlewares.

File: docs/third-party-packages.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ Starlette has a rapidly growing community of developers, building tools that int
 
 Here are some of those third party packages:
 
-
```
Comment (adriangb) [inline docs/third-party-packages.md:6]: Should we maybe be using a markdown linter / formatter to control newlines and stuff like that?

**Review by adriangb — APPROVED** [review]

File: docs/third-party-packages.md (hunk 1 above)
Comment (aminalaee) [reply docs/third-party-packages.md:6]: I think it would be great if we find one, otherwise we should just do it in PR review.

Comment (lovelydinosaur) [reply docs/third-party-packages.md:6]: +1 for just doing it in PR review.
Not a big deal either ways, and it's not worth the extra tooling.

**Pull request merged**